substra/sdk/rest_client.py

```diff
@@ -35,10 +35,14 @@ def __init__(self, config=None):
             self.set_config(config)
 
     def login(self):
+        headers = {
+            'Accept': self._headers['Accept'],
+        }
```

Comment:
Could you include a comment explaining that we don't use self._headers directly so that existing tokens aren't sent alongside the required Accept header?

Reply:
Yes of course! 🙂